### ISSUE TITLE
Kerberize cluster

### ISSUE THREAD

**Issue by joseluisvf** [joseluisvf/SEBC#13]

**Comment by joseluisvf** [comment]
Did this until the point where I was supposed to run the terasort command.

**Comment by joseluisvf** [comment]
I only got up to this point for the following reasons:

I installed scm-server on one node. scm-agent on all nodes.
Successfully setup the cluster.
Before going into HUE, I wanted to re-run the installation wizard in order to enable encryption.
However, on the rerun wizard, it couldnt get the heartbeats. Tried restarting the cloudera management service to no avail. Tried destroying it and booting it up again but i couldnt test the DB (which worked) due to the host not being in good health
We then trashed the cluster and tried booting it again, but the same problem persisted on the step for creating the databases.
Advised by Michael to kill instances and do everything from scratch.
Asked if I could ignore the GIT steps up until now, got an affirmative answer.
Asked if I was allowed to enable KERBEROS 5 minutes from the due time and got an affirmative answer.
In essence, I got to this point in 1 hour, plus the initial setup.

I apologize for the lack of readability for the pushes I made (after having had to thrash the instances). They required me to enter "    " before most lines in order for it to be pretty but I simply hadn't the time.
